keras/layers/convolutional.py

```diff
@@ -226,6 +241,23 @@ def get_config(self):
                 "ignore_border": self.ignore_border,
                 "stride": self.stride}
 
+class UpSample2D(Layer):
+    def __init__(self, upsample_size=(2, 2)):
+        super(UpSample2D,self).__init__()
+        self.input = T.tensor4()
+        self.upsample_size = upsample_size
+
+
+    def get_output(self, train):
+        X = self.get_input(train)
+        Y = theano.tensor.extra_ops.repeat(X, self.upsample_size[0], axis = -2)
```

Comment:
No particular reason, just my own style preference. Would you like me to change that?